common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -150,6 +151,7 @@ def is_api(auth_entry):
 
     AUTH_ENTRY_LOGIN_API,
     AUTH_ENTRY_REGISTER_API,
+    AUTH_ENTRY_SUDO
```

Comment:
@bradenmacdonald Can we remove frozenset and use only set to do the same thing like pipeline?